### DISCUSSION TITLE
What is the purpose of sending trpcState: `helpers.dehydrate()` as return props of `getStaticProps`?

### DISCUSSION THREAD

**Discussion by Senbonzakura1234** [trpc/trpc#4615] [Q&A]
I check [trpc ssg example](https://trpc.io/docs/client/nextjs/ssg) and `helpers.dehydrate()` was sent along side with id param to the component as props but never used in the component. What is the purpose of this?

**Comment by KATT** [comment]
It makes sure that data is in the react query cache as soon as the component loads in the browser
